CSV/specifications.csv

```diff
@@ -78,6 +79,7 @@ samsung,Samsung,"<a href='http://www.samsung.com/'>Samsung</a> "
 SDV,SDV,"<a href='http://www.sdcard.org/'>The SD Card Association</a>: SD Memory Card Specifications, Part 8, VIDEO SPECIFICATIONS"
 smptevc1,SMPTE,"<a href='http://www.smpte.org/'>SMPTE</a> RP2025:2007, VC-1 Bitstream Storage in the ISO Base Media File Format"
 Sony,Sony,<a href='http://www.sony.net'>Sony Corporation</a>
+SurveyApp, ISO/IEC 23000-10, ISO Surveillance application format
```

Comment:
`SurveyApp` is a weird acronym for an MPEG standard. Can't we use something like MPEG-VSAF ?

Reply:
Agreed. I changed it.